### PR TITLE
Custom types 5

### PR DESCRIPTION
**Pull request by destogl**

### CODE REVIEW

File: opcua/common/node.py [hunk 1]
```diff
@@ -292,6 +292,20 @@ def get_properties(self):
...
+    def get_variables(self):
+        """
+        return variables of node.
+        properties are child nodes with a reference of type HasComponent and a NodeClass of Variable
+        """
+        return self.get_children(refs=ua.ObjectIds.HasComponent, nodeclassmask=ua.NodeClass.Variable)
+
+    def get_methods(self):
+        """
+        return variables of node.
```
Comment (oroulet) [inline opcua/common/node.py]: Return methods :-)

Comment (destogl) [inline opcua/common/node.py]: Oh yes! Thanks!!

**Comment by oroulet** [comment]
I do not agree with everything but since this also contains good fixes,i think it is easier to merge then fix later

**Comment by oroulet** [comment]
Cannot merge... Conflicts

**Comment by destogl** [comment]
I will merge manually

**Comment by oroulet** [comment]
@destogl Can you merge today? I have some time today and I will try to have a look at this issue. I can also do the merge if you are busy

**Comment by destogl** [comment]
Merged. Sorry I came a bit later in the office today

**Pull request merged**